domain_access/domain_access.module

```diff
@@ -5,12 +5,13 @@
  * Domain-based access control for content.
  */
 
-use Drupal\domain\DomainNegotiator;
+use Drupal\domain\DomainManager;
```

Comment:
This is still written against the wrong branch. Development is done against domain 8.0.x branch, which changed to using DomainNegotiator a long time ago.